common/djangoapps/rocket_chat/views.py

```diff
@@ -0,0 +1,111 @@
+from django.conf import settings
+from django.http import Http404
+
+from edxmako.shortcuts import render_to_response
+from opaque_keys.edx.keys import CourseKey
+
+from courseware.access import has_access
+from courseware.courses import get_course_with_access
+from student.models import CourseEnrollment, anonymous_id_for_user, get_user
+
+from xmodule.modulestore.django import modulestore
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+
+from rocketchat_API.rocketchat import RocketChat as ApiRocketChat
```

Comment:
Yes, you are right, this was also a test of compatibility, because the API is for python 3 and for the xblock there are methods the API does not have, I think the better is extend this API on the xblock in order to reduce code and centralize an API.

Reply:
The test of compatibility you mention is checking if this python 3 package does not fail under our case of use? could it be a bit dangerous in the case we find a failure?